### PR TITLE
Add preview docs for dev-foo modules

### PR DESCRIPTION
**Pull request by tcdavis**
# Why

To document the current install process for dev-menu and dev-client, and provide a base for improving the dev functionality

### CODE REVIEW

File: docs/pages/client/distribution-for-ios.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+If you would prefer to install via the command line you can do so via `xcrun simctl install booted [path to .app folder]`
+
+## If you are installing on a physical device
+
+> 💡 The process for installing a development client build is currently **very manual**.  We are hard at work to provide a smoother flow for developers utiliziing our build service.
+
+
+### 1. Collect the UDID for any devices that will need to load the application
+
+Getting the UDID from non-developers is not a fool proof process.  [These](https://help.apple.com/xcode/mac/current/#/dev93ef696c6?sub=devdfa32588f) instructions should help.
```
Comment (lukmccall) [inline docs/pages/client/distribution-for-ios.md:42]:
```suggestion
Getting the UDID from non-developers is not a foolproof process. [These](https://help.apple.com/xcode/mac/current/#/dev93ef696c6?sub=devdfa32588f) instructions should help.
```

I'm not sure if the foolproof is a one-word or two 😅

File: docs/pages/client/distribution-for-ios.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+### 2. Register the target devices with an Ad-hoc Distribution Profile 
+
+Your target devices will need to be registered to an Ad-hoc Distribution Profile.  You can find the latest instructions for how to do so [here](https://help.apple.com/developer-account/#/devebd34abb1)
```
Comment (lukmccall) [inline docs/pages/client/distribution-for-ios.md:46]:
```suggestion
Your target devices will need to be registered to an Ad-hoc Distribution Profile. You can find the latest instructions for how to do so [here](https://help.apple.com/developer-account/#/devebd34abb1)
```
🧹

File: docs/pages/client/distribution-for-ios.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+You can generate a simulator build with `xcodebuild -sdk iphonesimulator -workspace ios/[MyApp].xcworkspace -scheme [MyApp]`
+
+### 2. Transfer Development IPA to other Developers
+
+After running the command, the generated APK will be named **MyApp.app** and can be found under `ios/build/Release-iphonesimulator/`
+
+<br /> 
+
+You can transfer the app folder via email, shared network directory, or however you would transfer an arbitrary file.  You may want to compress the .app folder to faciliate the transfer.
```
Comment (lukmccall) [inline docs/pages/client/distribution-for-ios.md:26]:
```suggestion
You can transfer the app folder via email, shared network directory, or however you would transfer an arbitrary file. You may want to compress the .app folder to facilitate the transfer.
```

File: docs/pages/client/distribution-for-ios.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+### 4. Install on device
+
+**to install via itunes**
+
+[godspeed](https://developer.apple.com/forums/thread/86806)
+
+**to install via download**
+
+You can serve the IPA according to the instructions [here](https://docs.monaca.io/en/products_guide/monaca_ide/deploy/non_market_deploy/) under install using OTA Deployment to allow any of your target users / devices to run your IPA afer downloading it from a public URL
```
Comment (lukmccall) [inline docs/pages/client/distribution-for-ios.md:60]:
```suggestion
You can serve the IPA according to the instructions [here](https://docs.monaca.io/en/products_guide/monaca_ide/deploy/non_market_deploy/) under install using OTA Deployment to allow any of your target users / devices to run your IPA after downloading it from a public URL
```

File: docs/pages/client/coming-from-react-native.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
+---
+title: Coming from React Native
+---
+
+import InstallSection from '~/components/plugins/InstallSection';
+
+import ConfigurationDiff from '~/components/plugins/ConfigurationDiff';
+
+
+**Expo Dev Client** is an open source project to improve your development experience while working on your Expo and React Native projects. It allows your team to focus on the JavaScript portion of your project, only needing to interact with XCode or Android Studio when you need to make changes to the native code used in your project .
```
Comment (lukmccall) [inline docs/pages/client/coming-from-react-native.md:10]:
```suggestion
**Expo Dev Client** is an open source project to improve your development experience while working on your Expo and React Native projects. It allows your team to focus on the JavaScript portion of your project, only needing to interact with XCode or Android Studio when you need to make changes to the native code used in your project.
```
🧹

File: docs/pages/client/installation.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
+---
+title: Installation
+---
+
+import InstallSection from '~/components/plugins/InstallSection';
+
+import ConfigurationDiff from '~/components/plugins/ConfigurationDiff';
+
+
+**Expo Dev Client** is an open source project to improve your development experience while working on your Expo and React Native projects. It allows your team to focus on the JavaScript portion of your project, only needing to interact with XCode or Android Studio when you need to make changes to the native code used in your project .
```
Comment (lukmccall) [inline docs/pages/client/installation.md:10]:
```suggestion
**Expo Dev Client** is an open source project to improve your development experience while working on your Expo and React Native projects. It allows your team to focus on the JavaScript portion of your project, only needing to interact with XCode or Android Studio when you need to make changes to the native code used in your project.
```
🧹

File: docs/pages/client/installation.md [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+
+> ⚠️ **Managed Expo projects are not yet supported**, but we are working on bringing it to EAS Build! If you want to build a managed Expo project with the Development Client, you'll have to eject it first. See the [Ejecting to Bare Workflow](../../workflow/customizing/) page to learn how.
+
+
+## Dream Instructions
+
+### The easiest way to get started is to initialize a new project.
+All of our default templates start with expo-dev pre installed
```
Comment (lukmccall) [inline docs/pages/client/installation.md:18]:
```suggestion

All of our default templates start with expo-dev pre installed.
```

File: docs/pages/client/installation.md [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+
+## 3. Install expo-dev-client
+
+The Dev Client module provides an easy way to connect your built application to a local bundler, allowing your team to work on the javascript portion of your application without ever having to load XCode or Android Studio.
+
+### CLI
+<InstallSection packageName="expo-development-client" cmd={["npm install expo-development-client", "npm install expo-barcode-scanner", "npm install expo",  "npx pod-install"]} hideBareInstructions />
+
+### Additional Steps
+On Android, you need to go in to the OS and manually grant yourself the camera permission
```
Comment (lukmccall) [inline docs/pages/client/installation.md:72]:
```suggestion

On Android, you need to go into the OS and manually grant yourself the camera permission
```

File: docs/pages/client/distribution-for-ios.md (hunk 1 above)
Comment (tcdavis) [reply docs/pages/client/distribution-for-ios.md:42]: 🔥  stupid native english speaker over here 😄

**Pull request merged**